build.sh

```diff
@@ -59,10 +79,11 @@ cd python3.*
 rm -rf test ensurepip idlelib
 cd lib-dynload
 rm -f _test* _tkinter*
-    
+
 echo "Copying certificates..."
 
-certs="$(python -c 'import certifi; print(certifi.where())')"
+pip3 install certifi --user
```

Comment:
No auto installing Python packages please. Let the user do this.

Reply:
👍 yeah I started by JUST making the build.sh work on macOS as is.  Then I thought well if it was ran in docker (where the pip3 install SHOULD happen) then it would work on all OSes and FINALLY thought well you could just build it with Github Actions and then everyone could see it being built and verify it was legit by the sha512 hash as the best solution (as you mention people probably won't run the installer builder and will download the installer)